kernel/nativelib.ml

```diff
@@ -158,7 +158,7 @@ let call_linker ?(fatal=true) prefix f upds =
    with Dynlink.Error e as exn ->
      let exn = CErrors.push exn in
      let msg = "Dynlink error, " ^ Dynlink.error_message e in
-     if fatal then (Feedback.msg_error (Pp.str msg); iraise exn)
+     if fatal then (Feedback.msg_warning (Pp.str msg); iraise exn)
```

Comment:
This should not be made a warning-level printing. From what I understand from this PR, this data (`msg`) should be part of the exception. I'm not sure how that can be done, since we don't know the exception statically. But I'll have to think more. Probably it is the handler logic that should be fixed to print the right message.

Reply:
I wish there was a simple fix to this, but a proper fix is involved due to the complicated state of error handling/printing system. This PR is pragmatic: at least output something, vs the current state of not outputting anything.

If the STM / protocol was not broken in that regard indeed we could use `msg_error`, but that is not the case.